website/pages/en/deploying/multiple-networks.mdx

```diff
@@ -0,0 +1,240 @@
+```
+
+### Using subgraph.yaml template
+
+One solution for older graph-cli versions that allows to parameterize aspects like contract addresses is to generate parts of it using a templating system like [Mustache](https://mustache.github.io/) or [Handlebars](https://handlebarsjs.com/).
+
+To illustrate this approach, let's assume a subgraph should be deployed to mainnet and Sepolia using different contract addresses. You could then define two config files providing the addresses for each network:
+
+```json
+{
+  "network": "mainnet",
+  "address": "0x123..."
+}
+```
+
+and
+
+```json
+{
+  "network": "sepolia",
+  "address": "0xabc..."
+}
+```
+
+Along with that, you would substitute the network name and addresses in the manifest with variable placeholders `{{network}}` and `{{address}}` and rename the manifest to e.g. `subgraph.template.yaml`:
+
+```yaml
+# ...
+dataSources:
+  - kind: ethereum/contract
+    name: Gravity
+    network: mainnet
+    network: {{network}}
+    source:
+      address: '0x2E645469f354BB4F5c8a05B3b30A929361cf77eC'
+      address: '{{address}}'
+      abi: Gravity
+    mapping:
+      kind: ethereum/events
+```
+
+In order to generate a manifest to either network, you could add two additional commands to `package.json` along with a dependency on `mustache`:
+
+```json
+{
+  ...
+  "scripts": {
+    ...
+    "prepare:mainnet": "mustache config/mainnet.json subgraph.template.yaml > subgraph.yaml",
+    "prepare:sepolia": "mustache config/sepolia.json subgraph.template.yaml > subgraph.yaml"
+  },
+  "devDependencies": {
+    ...
+    "mustache": "^3.1.0"
+  }
+}
+```
+
+To deploy this subgraph for mainnet or Sepolia you would now simply run one of the two following commands:
+
+```sh
+# Mainnet:
+yarn prepare:mainnet && yarn deploy
+
+# Sepolia:
+yarn prepare:sepolia && yarn deploy
+```
+
+A working example of this can be found [here](https://github.com/graphprotocol/example-subgraph/tree/371232cf68e6d814facf5e5413ad0fef65144759).
+
+**Note:** This approach can also be applied to more complex situations, where it is necessary to substitute more than contract addresses and network names or where generating mappings or ABIs from templates as well.
+
+## Checking subgraph health
+
+If a subgraph syncs successfully, that is a good sign that it will continue to run well forever. However, new triggers on the network might cause your subgraph to hit an untested error condition or it may start to fall behind due to performance issues or issues with the node operators.
+
+Graph Node exposes a graphql endpoint which you can query to check the status of your subgraph. On a local node, it is available on port `8030/graphql` by default. The full schema for this endpoint can be found [here](https://github.com/graphprotocol/graph-node/blob/master/server/index-node/src/schema.graphql). Here is an example query that checks the status of the current version of a subgraph:
+
+```graphql
```

Comment:
imo this just adds confusion for end users. This is a thing they can only really query on the local. On the network it isn't easy to access this data

Reply:
I was asked by the support team to include this material since it is helpful in certain cases. Thank you for flagging!